rustls/src/record_layer.rs

```diff
@@ -204,6 +206,13 @@ impl RecordLayer {
         self.write_seq
     }
 
+    /// Returns the number of remaining write sequences
```

Comment:
opted for not doing this since adding it on its own would require putting an `#[allow(dead_code]` attribute on it that gets removed in the immediate commit